sktime/forecasting/compose/_pipeline.py

```diff
@@ -1291,11 +1309,12 @@ class ForecastX(BaseForecaster):
     def __init__(
         self,
         forecaster_y,
-        forecaster_X,
+        forecaster_X=None,
```

Comment:
Should/can we test for this, that not passing `forecaster_X` defaults to `forecaster_y`?

Reply:
it is tested that it runs, by one of the `get_test_params` cases